### DISCUSSION TITLE
DB schema not created with additional fields

### DISCUSSION THREAD

**Discussion by bert2002** [fastapi-users/fastapi-users#1102] [Q&A] [answered]
## Describe the bug

The `User` table does not create additional columns.

## To Reproduce

```
class User(SQLAlchemyBaseUserTableUUID, Base):
  owner_id: int

class UserRead(schemas.BaseUser[uuid.UUID]):
  owner_id: int

class UserCreate(schemas.BaseUserCreate):
  pass

class UserUpdate(schemas.BaseUserUpdate):
  owner_id: int
```

at startup we create the tables

```
@app.on_event("startup")
async def on_startup():
  await create_db_and_tables() # fastapi-users
```

but the new column `owner_id` is not created

```
MariaDB [db]> describe user;
+-----------------+---------------+------+-----+---------+-------+
| Field           | Type          | Null | Key | Default | Extra |
+-----------------+---------------+------+-----+---------+-------+
| email           | varchar(320)  | NO   | UNI | NULL    |       |
| hashed_password | varchar(1024) | NO   |     | NULL    |       |
| is_active       | tinyint(1)    | NO   |     | NULL    |       |
| is_superuser    | tinyint(1)    | NO   |     | NULL    |       |
| is_verified     | tinyint(1)    | NO   |     | NULL    |       |
| id              | char(36)      | NO   | PRI | NULL    |       |
+-----------------+---------------+------+-----+---------+-------+
6 rows in set (0.003 sec)

MariaDB [db]> 

```

## Expected behavior

I would expect the `User` table to have the column `owner_id` .

## Configuration
 - Python version : `3.10.6`
 - FastAPI version : `0.85.1`
 - FastAPI Users version : `10.2.0`

**Comment by frankie567** [comment, marked as answer]
Did your table was created before you added the column? Is so, it's expected: `create_db_and_tables` is not able to modify existing tables. Try to start from an empty database and it should correctly create all the columns.

If you need a proper migration system for your SQL database, you should have a look at Alembic: https://alembic.sqlalchemy.org/en/latest/

**Reply by bert2002** [reply]
I dropped all tables and they are getting recreated, but not with the additional field.Is there a way to increase the debugging?
I use alembic for other tables, but would hope that I can get `create_db_and_tables` working with its defaults.

**Reply by frankie567** [reply]
Ha sorry, I didn't spot what you missed here. In your `User` model, you need to actually define your `owner_id` column (type, constraints, etc). Adding the property alone is not enogu for SQAlchemy.

See: https://docs.sqlalchemy.org/en/14/orm/quickstart.html#declare-models

**Reply by bert2002** [reply]
Of course and it makes sense. Was mixing `User` and the other classes. Very good hint :)

**Comment by bete7512** [comment]
also I have got same issue 

```
class User(Base,SQLAlchemyBaseUserTableUUID):
    __tablename__ = "users"    
    first_name = Column(String, nullable=False)
    phone_number = Column(String, nullable=True)
```
